ruby_on_rails/README.md

```diff
@@ -0,0 +1,184 @@
+  1. [Ruby User's Guide](http://www.rubyist.net/~slagell/ruby/)
+  1. [Ruby Web Dev: The Other Way](https://leanpub.com/rwdtow/read)
+  1. [The Bastards Book of Ruby](http://ruby.bastardsbook.com/)
+  1. [The Unofficial Ruby Usage Guide (at Google) - Ian Macdonald](http://www.caliban.org/ruby/rubyguide.shtml)
+  1. [Why's (Poignant) Guide to Ruby](http://poignant.guide/)
+
+Reference: https://github.com/EbookFoundation/free-programming-books/blob/master/free-programming-books.md#ruby
+
+#### Ruby on Rails:
+  1. [A community-driven Rails style guide](https://github.com/bbatsov/rails-style-guide)
+  1. [Building REST APIs with Rails](http://apionrails.icalialabs.com/book)
+  1. [Kestrels, Quirky Birds, and Hopeless Egocentricity](https://leanpub.com/combinators/read)
+  1. [Learn Ruby on Rails as You Modify a Craigslist Clone](http://www.thinkful.com/learn/ruby-on-rails-tutorial/)
+  1. [Objects on Rails](http://objectsonrails.com/)
+  1. [Rails Girls Guides](http://guides.railsgirls.com/)
+  1. [Ruby on Rails 3.2 - Step by Step](http://www.xyzpub.com/en/ruby-on-rails/3.2/)
+  1. [Ruby on Rails 4.0 Guide](http://www.xyzpub.com/en/ruby-on-rails/4.0/)
+  1. [Ruby on Rails Guides](http://guides.rubyonrails.org/)
+  1. [Ruby on Rails Tutorial (Rails 5): Learn Web Development with Rails - Michael Hartl](https://www.railstutorial.org/book)
+  1. [Upgrading to Rails 4](https://github.com/alindeman/upgradingtorails4)
+
+Reference: https://github.com/EbookFoundation/free-programming-books/blob/master/free-programming-books.md#ruby-on-rails
+
+## Style Guides
+  - [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide)
+  - [Rails Style Guide](https://github.com/bbatsov/rails-style-guide)
+
+## Additional Resources
+#### Screencasts:
+  1. [Go Rails](https://gorails.com/episodes)
+  1. [RailsCasts](http://railscasts.com/)
+  1. [Drifting Ruby](https://www.driftingruby.com/)
+
+#### Free Courses:
+  1. [edX - Agile Development Using Ruby on Rails](https://www.edx.org/professional-certificate/agile-development-using-ruby-rails)
+  1. [Udemy - Ruby on Rails a Beginners Guide](https://www.udemy.com/ruby-on-rails-a-beginners-guide-free/)
+  1. [Udemy - Learn Ruby on Rails from Scratch](https://www.udemy.com/learn-ruby-on-rails-from-scratch/)
+  1. [CodeCademy - Learn Ruby on Rails](https://www.codecademy.com/learn/learn-rails)
+
+#### Paid Courses:
+  1. [Udemy - The Complete Ruby on Rails Developer Course](https://www.udemy.com/the-complete-ruby-on-rails-developer-course/)
+  1. [Coursera - Ruby on Rails Web Development Specialization ](https://www.coursera.org/specializations/ruby-on-rails)
+  1. [Lynda - Ruby on Rails Training and Tutorials](https://www.lynda.com/Ruby-on-Rails-training-tutorials/304-0.html)
+  1. [Team Tree House - Rails Development](https://teamtreehouse.com/tracks/rails-development)
+  1. [ONE MONTH RAILS](https://onemonth.com/courses/rails)
+
+#### Tutorials I Have Written:
```

Comment:
`Tutorials I have written` doesn't make sense for a training guide under `10Pearls` name. You can simple move these under helpful links/resources section.